README.md

```diff
@@ -29,11 +40,13 @@ Browse to: `http://localhost:8080/` to login!
 
 ### `custom-login`
 
+The login is achieved with the [Okta Sign In Widget][], which gives you more control to customize the login experience within your application.  After the user authenticates, they are redirected back to the application.
+
 ```bash
 cd custom-login
-mvn -Dokta.oauth2.issuer=https://dev-123456.oktapreview.com/oauth2/default \
-    -Dokta.oauth2.clientId=YourClientId \
-    -Dokta.oauth2.clientSecret=YourClientSecret
+mvn -Dokta.oauth2.issuer=https://{yourOktaDomain}.com/oauth2/default \
```

Comment:
I don't think this is clear hear, as unlike our blog this property will not be filtered. I think leaving it as dev-12345.oktapreview is easier to understand (and may help remove some confusion with the`-admin` url)